lib/report/cost_report.rb

```diff
@@ -65,6 +65,15 @@ def freq_table
       @freq_table
     end
 
+    # Dump freq table so these computes can be re-used in member_counts_report.
```

Comment:
Should `class FrequencyTable` be a thing? Then feed freq_table to CostReport or MemberCountsReport. Possibly a refactor for another day.

Reply:
Now a jira issue to talk about at a later date. https://tools.lib.umich.edu/jira/browse/HT-3165